karma.conf.js

```diff
@@ -0,0 +1,68 @@
+module.exports = function(config) {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+    // list of files / patterns to load in the browser
+    files: [
+      { pattern: 'test/*.js', type: 'module' }
+    ],
+    exclude: [],
+
+    plugins: [
+      // load plugin
+      require.resolve('@open-wc/karma-esm'),
+
+      // fallback: resolve any karma- plugins
+      'karma-*',
+    ],
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['esm', 'jasmine', 'detectBrowsers'],
```

Comment:
I'd slightly prefer to keep using mocha and chai as we do in the other projects. They work fine with Karma in my experience.

Reply:
I just think that since basically every single one of our projects uses that that we should be consistent until we decide to do change what we're going to do going forward for all projects, then migrate the other active projects over time.

Reply:
I hear you. I don't think this should be blocking. It's an entirely internal detail, so we're free to revisit at any time